libs/e2e/src/lib/describer/modelAssessment/modelOverview/ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent.ts

```diff
@@ -16,14 +16,18 @@ export function ensureAllModelOverviewDatasetCohortsViewBasicElementsArePresent(
   includeNewCohort: boolean,
   isNotebookTest: boolean
 ): void {
+  const isVision =
```

Comment:
why is this condition and the above condition different?
    const isVision =
      datasetShape.isObjectDetection ||
      datasetShape.isMultiLabel ||
      datasetShape.isImageClassification
        ? true
        : false;

Reply:
Good question, it's because it's value with the condition could also be undefined, hence I'm pegging it to a boolean here.